Texconv/texconv.cpp

```diff
@@ -184,6 +185,7 @@ const SValue g_pOptions[] =
     { L"c",             OPT_COLORKEY },
     { L"tonemap",       OPT_TONEMAP },
     { L"x2bias",        OPT_X2_BIAS },
+    { L"presalphacov",  OPT_PRESERVE_ALPHA_COVERAGE },
```

Comment:
Maybe ``keepcoverage`` would be easier to parse?

Reply:
Fair point, I couldn't decide on a good command name. "keepcoverage" would work fine.